requirements/requirements-dev.txt

```diff
@@ -70,7 +70,7 @@ soupsieve==1.8
 spotipy==2.4.4
 tqdm==4.31.1
 twitterapi==2.5.9
-typed-ast==1.3.1
+typed-ast==1.2.0
```

Comment:
That's odd. pip-compile does not forward the environment marker from `astroid`s `install_requires` [1].

[1] https://github.com/PyCQA/astroid/blob/8a3d8b908da5d80c7bf6857d94333e7c87b5c176/astroid/__pkginfo__.py#L33